optax/_src/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/home/joaogui/miniconda3/envs/optax/bin/python"
```

Comment:
Can you remove this file?

We added a .gitignore now that should avoid importing this